src/processing/mode/android/SDKDownloader.java

```diff
@@ -32,7 +33,7 @@
 @SuppressWarnings("serial")
 public class SDKDownloader extends JFrame implements PropertyChangeListener {
 
-  private static final String URL_REPOSITORY = "https://dl-ssl.google.com/android/repository/repository-10.xml";
+  private static final String URL_REPOSITORY = "https://dl-ssl.google.com/android/repository/repository-11.xml";
```

Comment:
Not sure if this is necessary. 
But I realized Android SDK manager downloads the xml from this newer url.
I did a git diff and there was some difference between the two. But the part with which we are concerned is same in both xml.